core/templates/dev/head/components/activityTilesInfinityGrid.js

```diff
@@ -23,7 +23,7 @@ oppia.directive('activityTilesInfinityGrid', [function() {
     controller: [
       '$scope', '$rootScope', 'searchService',
       function($scope, $rootScope, searchService) {
-        $scope.showMoreExplorations = function() {
+        $scope.showMoreActivities = function() {
```

Comment:
Does this function (and the corresponding ``searchService``) need to be updated to support collections?